db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+user = User.create! :name => 'Admin', :email => 'admin@motohelper.pl', :password => 'mh1234', :password_confirmation => 'mh1234'
```

Comment:
Don't use old hash syntax. 
Have you run ``` overcommit --install ``` in your project directory. There are many rubocop issues
```
Inspecting 36 files
......CC..C...C.......C..........C..

Offenses:

spec/factories/users.rb:1:1: C: Missing magic comment # frozen_string_literal: true.
FactoryBot.define do
^
spec/factories/users.rb:3:1: C: Trailing whitespace detected.
spec/models/user_spec.rb:1:1: C: Missing magic comment # frozen_string_literal: true.
require 'rails_helper'
^
app/controllers/registrations_controller.rb:1:1: C: Missing magic comment # frozen_string_literal: true.
class RegistrationsController < Devise::RegistrationsController
^
app/controllers/registrations_controller.rb:2:1: C: Extra empty line detected at class body beginning.
app/models/user.rb:1:1: C: Missing magic comment # frozen_string_literal: true.
class User < ApplicationRecord
^
app/models/user.rb:6:1: C: Extra empty line detected at class body end.
config/initializers/devise.rb:1:1: C: Missing magic comment # frozen_string_literal: true.
# Use this hook to configure devise mailer, warden hooks and so forth.
^
config/initializers/devise.rb:9:121: C: Line is too long. [154/120]
  # config.secret_key = '9bb0509b91ebd0621604af527b033ae249e8fbb120bc85985c46238998ec4ca3b035fb29869e41a6f3670e34bf679e1241ef7dec049d7dfbd81dea615342c97a'
                                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
config/initializers/devise.rb:111:121: C: Line is too long. [150/120]
  # config.pepper = 'deb1286bda6bbfc375dbdb4d8a5acd67474dcbe78aee1e34d37919daf0ce19941209b22f715841350171a8e6fa60bfdf2732deb44d1eb23a87b69274bdfc8a47'
                                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
config/routes.rb:7:22: C: Use the new Ruby 1.9 hash syntax.
  devise_for :users, :controllers => { registrations: 'registrations' }
                     ^^^^^^^^^^^^^^^
config/routes.rb:8:1: C: Extra empty line detected at block body end.

36 files inspected, 12 offenses detected
```